### PR TITLE
Create admin sets

### LINKED ISSUE #2500: An admin should be able to create admin sets.
- [x] There should be a page from the admin menu to create admin sets
- [x] There should be fields for title, description, <strike>language</strike>*
- [x] It should record the date created and <strike>language</strike>**
- [ ] <strike>All admin sets should be created with public read and admin group edit access.</strike>***
- [x] Only admins can create admin sets.

\* language moved to #2579
\*\* creator moved to #2581
\*\*\* Access controls move to #2582

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #2500 

@projecthydra/sufia-code-reviewers

screen shot 2016-08-29 at 5 19 05 pm
screen shot 2016-08-29 at 5 19 20 pm

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@jcoyne Can you include a screenshot of the UI changes in this PR?

**Comment by jcoyne** [comment]
@mjgiarlo I don't know why coveralls thinks coverage dropped here. Any ideas?

File: app/controllers/sufia/admin/admin_sets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module Sufia
+  class Admin::AdminSetsController < ApplicationController
+    load_and_authorize_resource
+    layout 'admin'
+
+    def index
+      authorize! :manage, AdminSet
+      add_breadcrumb  'Home', root_path
```
Comment (awead) [inline app/controllers/sufia/admin/admin_sets_controller.rb]: i18n-ify?

File: app/views/layouts/admin.html.erb [hunk 2]
```diff
@@ -36,6 +36,10 @@
                           <span class="fa fa-cog"></span> Settings
                         <% end %>
                     </li>
+                    <li><%= link_to sufia.admin_admin_sets_path do %>
+                          <span class="fa fa-sitemap"></span> Administrative Sets
```
Comment (awead) [inline app/views/layouts/admin.html.erb]: i18n

File: app/views/sufia/admin/admin_sets/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-sitemap"></span> Administrative Sets</h1>
```
Comment (awead) [inline app/views/sufia/admin/admin_sets/index.html.erb]: i18n here too, since it's the same.

File: app/views/sufia/admin/admin_sets/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                  <th>Creator</th>
+                  <th>Visibility</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td><a href="admin-set-show.html">Steven Chu Papers</a></td>
+                  <td>2013-06-12</td>
+                  <td>repo-manager@stanford.edu</td>
+                  <td>Public</td>
```
Comment (awead) [inline app/views/sufia/admin/admin_sets/index.html.erb]: why is this here?

File: app/views/layouts/admin.html.erb (hunk 2 above)
Comment (jcoyne) [inline app/views/layouts/admin.html.erb]: Will do in https://github.com/projecthydra/sufia/issues/2594

File: app/views/sufia/admin/admin_sets/index.html.erb (hunk 4 above)
Comment (jcoyne) [inline app/views/sufia/admin/admin_sets/index.html.erb]: Place holder for https://github.com/projecthydra/sufia/issues/2564
